src/main/js/EditorPage.jsx

```diff
@@ -395,7 +395,12 @@ class PipelineLoader extends React.Component {
                     this.pipelineIsModified = false;
                     this.lastPipeline = JSON.stringify(convertInternalModelToJson(pipelineStore.pipeline));
                     // If this is a save on the same branch that already has a Jenkinsfile, just re-run it
-                    if (this.state.sha && branch === body.content.branch) {
+//                    if (!repo) { // this is a multibranch, not org folder, run it to index
```

Comment:
I want this here to remind me to make a different change which I can't right now